src/test-utils.js

```diff
@@ -0,0 +1,28 @@
+import { getRpcClient } from "./rpc-client";
+import sinon from 'sinon';
+
+let request = getRpcClient().request;
+
+export function mockRpcClient() {
+    getRpcClient().request = jest.fn(() => Promise.resolve({}));
```

Comment:
can we mock out exceptions being thrown by the rpcClient too? It might be interesting to test error scenarios as well.

Reply:
Sure

Reply:
Added to https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871

`test error scenarios on json rpc calls`